paket.lock

```diff
@@ -1,10 +1,15 @@
 NUGET
   remote: http://www.nuget.org/api/v2
   specs:
+    Dynamitey (1.0.2)
     Expression.Blend.Sdk (1.0.2)
     Fantomas (2.1)
       FSharp.Compiler.Service (>= 2.0.0.8)
     FSharp.Compiler.Service (2.0.0.8)
+    FSharp.Core (4.0.0.1)
```

Comment:
@forki Regardless, I think I might just pull it off with the Reflection module @dungpa linked above, and no need for this dependency. I'll revert.